lib/cli.rs

```diff
@@ -253,6 +253,11 @@ pub struct WalletConfig {
     /// the wallet is large and periodic syncs are not feasible.
     #[arg(long = "wallet-skip-periodic-sync", default_value_t = false)]
     pub skip_periodic_sync: bool,
+
+    /// Path to a file containing exactly 12 space-separated BIP39 mnemonic words.
+    /// Cannot be used together with --wallet-auto-create.
+    #[arg(long = "set-seed-with-mnemonic")]
```

Comment:
I think a better name here would be `wallet-seed-file`. This flag sounds like a bool, and the other wallet flags start with `wallet-`